inc/knowbaseitem.class.php

```diff
@@ -1471,12 +1492,19 @@ static function showList($options, $type = 'search') {
                      $href = " href=\"".KnowbaseItem::getFormURLWithID($data["id"])."\" ";
                   }
 
+                  $fa_class = "";
+                  $fa_title = "";
+                  if ($data['visibility_count'] <= 0) {
+                     $fa_class = "fa-eye-slash not-published";
+                     $fa_title = __s("This item is not published yet");
+                  }
+                  if ($data['is_faq']) {
```

Comment:
What if `$data['visibility_count'] <= 0 && $data['is_faq']`?

Reply:
no icon, intended

Reply:
-> faq icon
Seems to be ok